security/zynqmp/generate_keys.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Here for info purposes only.
+# We already have our keys in this repository
+#
+#if [ ! -f bootgen-spl ]; then
```

Comment:
Did you want to leave this commented (even if it's for info purposes)?

Reply:
yes, it is kind of dangerous unless you know what you are doing....